frontend/src/components/ArtifactList/index.js

```
@@ -0,0 +1,128 @@
+import {Component} from 'react'
+import {connect} from "react-redux";
+import {componentsOperations} from "../duck";
+import artifactList from "./presenter.jsx"
+import get from "lodash/get"
+import queryString from "query-string";
+import history from "../../utils/history";
+import {isPrintableArtifact} from "../common";
+
+const mapStateToProps = (state) => {
+
+    const adminMode = get(state, "components.adminMode", false)
```

ESLint

[error] Expected indentation of 2 spaces but found 4.

(indent)